git-keeper.py

```diff
@@ -148,7 +158,7 @@ def main():
             )
             if git_hub_private_repo_error_text not in str(e):
                 error = True
-                logging.error(e)
+                logger.exception("Found an error")
```

Comment:
Please use `logging.error` instead, `logging.exception` should be used in a `try: except` block.